config/helm/chart/default/templates/Common/crd/dynatrace-operator-crd.yaml

```diff
@@ -0,0 +1,4 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes, openshift, google" (include "dynatrace-operator.platformSet" .))}}
+{{ if eq (include "dynatrace-operator.partial" .) "false" }}
```

Comment:
missing `installCRD` flag to explicitly install CRD

Reply:
Dynatrace-operator helm chart purpose is now CRD management, isn't it?

Reply:
yes, but having the ability to opt out is much safer

Reply:
What's scenario for using dynatrace-operator helm chart with `installCRD=false`?